docs/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc

```diff
@@ -37,8 +37,8 @@ image:viewer1.png[Viewer tab]
 Select a variable in the table, right-click to open its context menu, and choose "NCdump Data".
 The "NCDump Variable Data" dialog will pop up. Here you can print (i.e. "dump") its values to the screen.
 
-Here�using link:http://www.adt.unipd.it/corsi/Bianco/www.pcc.qub.ac.uk/tec/courses/f90/stu-notes/F90_notesMIF_5.html#HEADING41[Fortran 90 array section syntax]
-(`start:end:stride`)�you can print (i.e. "dump") all or part of the variable's values to the screen. By default,
+Here�using link:http://www.adt.unipd.it/corsi/Bianco/www.pcc.qub.ac.uk/tec/courses/f90/stu-notes/F90_notesMIF_5.html#HEADING41[Fortran 90 array section syntax]
```

Comment:
Looks like some kind of encoding issue?

Reply:
Could be.  I’ll fix it. Give me a second.

Reply:
Could be.  I’ll fix it. Give me a second.